tests/unit_tests/cli/test_integration_cli.py

```diff
@@ -388,7 +388,7 @@ def test_that_prior_is_not_overwritten_in_ensemble_experiment(
     with tmpdir.as_cwd():
         ert = EnKFMain(ResConfig("poly_example/poly.ert"))
         prior_context = ert.load_ensemble_context(
-            "default", list(range(ert.getEnsembleSize())), 0
+            "default", [True] * ert.getEnsembleSize(), 0
```

Comment:
Long live duck typing? Technically then this test would have provided a list of values that are truthy except for the first falsy..

I'm just curious how this would be interpreted by `load_ensemble_context`.. And also, how this test would pass as you are running an ensemble experiment on line 411.